mentoring/processes/group-mentoring.md

```diff
@@ -1,23 +1,75 @@
-group-mentoring.md may have some other process info baked into the program info.
-
-## Logistics
-1. SIG/WG determines need
-2. Consults with sig-contribex; contribex sets up infra (slack channel; TODO build this out)
-3. Posts a note to their mailing list and k-dev that they are looking for more
-reviewers/approvers/x_role and would like to form a group to mentor who are
-already in the role below it. (example: already a member and they need more
-  reviewers)
-4. Sets a start date and end date; schedule a kick off meeting so everyone can
-meet
-over video/audio (1 and only unless the group establishes other comms). Also set
-the weekly slack standup days so they are consistent.
-5. Program begins
-6. If the mentors think certain skills will need to be taught/learned, they
-should be done upfront. (We scattered them for the first cohort and there were
-  things that would be been more valuable to learn in the beginning of the
-  journey)  
-6. Feedback is continuous*
-7. Mentors start PRs for those who graduated  
-
-*Feedback is continuous but will have a half way check in point. The goal is that
-mentees should not be surprised by the outcome of the program.
+# Running a Contributor Ladder Growth Program 
+
+There are differences in ciricuulum for reviewer and Chair and/or Tech Lead 
+groups but the foundation of each are mostly the same unless otherwise noted 
+below in the set up instructions. This difference is also reflected in our 
+[community membership guidelines].
+
+## Discovery of Needs
+
+Cadence:  
+ContribEx sets a quarterly issue in k/community requesting SIGs to participate 
```

Comment:
Suggest we do per-release rather than quarterly. Issue should probably precede the release by about 1mo.